styles.css

```diff
@@ -0,0 +1,971 @@
+/*Fonts*/
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Dancing+Script:wght@500&display=swap");
+
+/* Component Library Color styles */
+:root {
+	--text-color: #1f2937;
+	--text-dark-color: #f3f4f6;
+
+	--color-primary: #51c84d; /*#06b100; /* #33cccc; */
+	--color-primary-dark: #012300; /* #0a2929; */
+	--color-primary-bg: #b4e8b3; /* #d4ffff; */
+	/* --color-secondary: #3b82f6;
+	--color-secondary-dark: #1e3a8a;
+	--color-secondary-bg: #bfdbfe; */
+	--color-secondary: #7868e6;
+	--color-secondary-dark: #0c0a17;
+	--color-secondary-bg: #d7d2fc;
+	--color-danger: #b91c1c;
+	--color-danger-bg: #fee2e2;
+	--color-success: #0b6631;
+	--color-success-bg: #d1fae5;
+	--color-warning: #ff6f00; /*#f59e0b;*/
+	--color-warning-bg: #fef3c7;
+	--color-grey-darkest: #1f2937;
+	--color-grey-dark: #4b5563;
+	--color-grey: #9ca3af;
+	--color-grey-light: #f3f4f6;
+	--color-grey-bg: #e5e7eb;
+
+	--color-facebook: #4267b2;
+	--color-twitter: #1da1f2;
+	--color-google: #db4437;
+}
+
+* {
+	margin: 0;
+	padding: 0;
+	font-family: "Roboto", sans-serif;
+	/* color: white; */
+	box-sizing: border-box;
+}
+
+body > * {
+	color: var(--color-grey-darkest);
+	padding-top: 90px;
+	scroll-behavior: smooth;
+}
+
+.container-center {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	flex-wrap: wrap;
+	/* width: 100%; */
+}
+
+.container {
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap;
+}
+
+.container-column {
+	display: flex;
+	/* justify-content: center; */
+	align-items: center;
+	flex-direction: column;
+	width: 100%;
+	max-width: 800px;
+	/* min-height: 100vh; */
+}
+
+.container-column p{
+	margin: 0rem 1rem;
+}
+.container-space-around {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	flex-wrap: wrap;
+	width: 100%;
+}
+
+li {
+	list-style: none;
+}
+
+a {
+	text-decoration: none;
+	color: inherit;
+}
+
+button {
+	cursor: pointer;
+	text-align: center;
+	outline: none;
+}
+
+input {
+	outline: none;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+	margin: 0.5rem;
+}
+
+/*navbar CSS*/
+.navbar {
+	background: white;
+	padding: 0.5rem 1rem;
+	display: flex;
+	width: 100%;
+	height: 3rem;
+	box-shadow: 1px 2px 5px var(--color-grey-bg);
+	position: fixed;
+	top: 0;
+	left: 0;
+	z-index: 10;
+	box-sizing: border-box;
```

Comment:
As you have already declared `box-sizing` in your `*` global attribute, you again don't need to add it explicitly you can remove it where ever you have added.